pandas/tests/io/json/test_pandas.py

```diff
@@ -82,76 +80,79 @@ def setup(self, datapath):
         del self.tsframe
         del self.mixed_frame
 
-    def test_frame_double_encoded_labels(self):
+    def test_frame_double_encoded_labels(self, orient):
         df = DataFrame(
             [["a", "b"], ["c", "d"]],
             index=['index " 1', "index / 2"],
             columns=["a \\ b", "y / z"],
         )
 
-        assert_frame_equal(df, read_json(df.to_json(orient="split"), orient="split"))
-        assert_frame_equal(
-            df, read_json(df.to_json(orient="columns"), orient="columns")
-        )
-        assert_frame_equal(df, read_json(df.to_json(orient="index"), orient="index"))
-        df_unser = read_json(df.to_json(orient="records"), orient="records")
-        assert_index_equal(df.columns, df_unser.columns)
-        tm.assert_numpy_array_equal(df.values, df_unser.values)
+        result = read_json(df.to_json(orient=orient), orient=orient)
+        expected = df.copy()
 
-    def test_frame_non_unique_index(self):
-        df = DataFrame([["a", "b"], ["c", "d"]], index=[1, 1], columns=["x", "y"])
+        if orient == "records" or orient == "values":
+            expected = expected.reset_index(drop=True)
+        if orient == "values":
```

Comment:
is the "values" case new?

Reply:
... never mind

Reply:
Yea this is equivalent to the original case of `tm.assert_numpy_array_equal(df.values, df_unser.values)` but more explicit as to the object being tested and avoids the `.values` call